lib/client.ts

```diff
@@ -679,6 +679,24 @@ export class OAuth {
     return this.http.postForm(`${OAUTH_BASE_PREFIX}/revoke`, { access_token });
   }
 
+  public verifyAccessToken(access_token: string): Promise<{}> {
```

Comment:
There's response why `{}` ?

Reply:
I referenced revokeAccesstoken API method.
What do you want to return value?
Do it expect expires_in?